vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,173 @@
+
+Approximations of remaining uses:
+
+- **Todos and Agenda** (57%)
+- **Research** (47%)
+- **Meeting notes** (45%)
+- **Team knowledge base** (29%)
+- **Publishing** (22%)
+
+18% of respondents provided additional information under **Other**. Users want to use Dendron for so many things that it was rather difficult to create good categories for them.
+
+> The percentages are in relation to the respondents in **Other**
+
+- Student/Learning/Class/Academia Notes (4%)
+- Journaling (4%)
+- Content Creation and Non-fiction/Fiction Writing (3%)
+- Project / Software Documentation (2%)
+- System/Product Configuration and Design (2%)
+- CRM (1%)
+- Second Brain (1%)
+- Gaming Notes (D&D, RPGs, General) (1%)
+
+There were some smaller groups that stood out in the **Other** category, having one to three respondents:
+
+- Scripted/Automated documentation creation
+- Tracking, Reading, and Taking Notes on Books
+- Recipes / Cookbook
+- Drafting GitHub issues/MRs
+- Event planning
+- Presentations
+
+## Are users coming from any existing tools?
+
+![Bar graph displaying different tools used before or with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-previous-tools.png)
+
+New users were able to select all that applied. More users said they were coming from **No Previous Tool** (26%) than users who found themselves looking at Dendron after something else, with **OneNote** (25%) coming in a close second.
+
+Approximations of remaining previous (or parallel) tools:
+
+- **Notion** (23%)
+- **Google Keep** (19%)
+- **Obsidian** (18%)
+- **Evernote** (14%)
+- **Roam** (5%)
+- **Logseq** (5%)
+- **Foam** (5%)
+
+15% of respondents provided additional information under **Other**, providing a total of over **70 different tools and workflows.** Here are the top **Other** tools listed.
+
+> The percentages are in relation to the respondents in **Other**
+
+- Joplin (9%)
+- org-mode / org-roam / emacs (6%)
+- Bear (6%)
+- Apple Notes (4%)
+- Plain Markdown or text files (4%)
+- Trello / Confluence / Jira (4%)
+- Typora (3%)
+- Physical notes (2%)
+
+## Do users have plans on publishing their notes?
+
+![Bar graph displaying levels of interest in publishing with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-publishing-interest.png)
+
+New users were only able to select a single response, if they were to respond to this question at all. A majority of users were **Curious** (39%) about publishing, or otherwise already saying **Yes** (36%) to planning publishing. The remaining respondents said **No** (25%) and were not interested in Dendron's publishing capabilities.
+
+## Takeaways
+
+Users are drawn to Dendron for Personal Knowledge Management (PKM), task management, research, and more.
+
+With that in mind, Dendron and the Dendron community have been introducing more features and enhancements that continue to develop:
```

Comment:
this sentence feels like it ends in the middle. 

Also, prefer [[structured docs|dendron://private/area.growth.ref.writing-style-guide#structured-docs]] (add groupings)
```
The Dendron team has been acting on user feedback and shipped some of the following features:
- On PKM
    - feature1
    - feature2
- On Task Management
    ...
```